commitizen/changelog.py

```diff
@@ -48,11 +50,11 @@
     VersionProtocol = typing.Any
 
 
-def get_commit_tag(commit: GitCommit, tags: List[GitTag]) -> Optional[GitTag]:
+def get_commit_tag(commit: GitCommit, tags: list[GitTag]) -> GitTag | None:
```

Comment:
So the unions with `|` work well before python 3.10 with the future annotations? Interesting

Reply:
Yep, it works well with 3.7 and above 🎉